torch/onnx/symbolic_opset17.py

```diff
@@ -76,32 +76,24 @@ def stft(
     Hence, this function does not contain these two arguments.
     See torch.stft source code for more info.
 
-    Parameters
-    ----------
-    g : jit_utils.GraphContext
-        Graph to write the ONNX representation into
-    input : _C.Value
-        Input tensor for the transformation
-    n_fft : int
-        FFT size
-    hop_length : int, optional
-        Size of the hop. Defaults to floot(n_fft // 4)
-    win_length : int, optional
-        Size of the analysis window. Defaults to `n_fft`
-    window : _C.Value, optional
-        Analysis window. Defaults to a window of all ones.
-    normalized : bool, optional
-        Whether to return a normalized STFT.
-    onesided : bool, optional
-        Whether to return only half (+1) of the results, given the symmetry
-        of the STFT
-    return_complex : bool, optional
-        Whether to return the complex value (Note: Must be False or None)
-
-    Returns
-    -------
-    op
-        Operator for torch.stft associated with STFT (ONNX).
+    Args:
+        g (jit_utils.GraphContext): Graph to write the ONNX representation into
+        input (_C.Value): Input tensor for the transformation
+        n_fft (int): FFT size
+        hop_length (int, optional): Size of the hop. Defaults to
+            `floot(n_fft // 4)`
+        win_length (int, optional): Size of the analysis window.
+            Defaults to `n_fft`
+        window (_C.Value, optional): Analysis window. Defaults to a window of
+            all ones
+        normalized (bool, optional): Whether to return a normalized STFT
+        onesided (bool, optional): Whether to return only half (+1) of the
```

Comment:
nit: no need to annotate types in the docstring because they are annotated in the function signature already

Reply:
Done!